bitnami/keycloak/templates/keycloak-config-cli-configmap.yaml

```
@@ -0,0 +1,21 @@
+{{- if and .Values.keycloakConfigCli.enabled .Values.keycloakConfigCli.config (not .Values.keycloakConfigCli.existingConfigmap) }}
```

Following the previous suggestion:
```suggestion
{{- if (include "keycloak. keycloakConfigCli .createConfigmap" .) }}
```